src/winstanley/WdlSyntaxHighlighter.scala

```diff
@@ -55,7 +55,7 @@ object WdlSyntaxHighlighter {
     WdlTypes.COMMENT -> DefaultLanguageHighlighterColors.LINE_COMMENT,
     WdlTypes.IMPORT -> DefaultLanguageHighlighterColors.KEYWORD,
     WdlTypes.WORKFLOW -> DefaultLanguageHighlighterColors.KEYWORD,
-    WdlTypes.WORKFLOW_IDENTIFIER_DECL -> DefaultLanguageHighlighterColors.FUNCTION_DECLARATION,
+    WdlTypes.IDENTIFIER -> DefaultLanguageHighlighterColors.FUNCTION_DECLARATION,
```

Comment:
Oh, now I see why we have this, it's so we can syntax highlight the different identifiers in different ways. It only shows up if you switch intelliJ to `Darcula` but it makes a huge difference so we should probably keep it like it was before.

Reply:
If we make the alternative `WdlCallableReference` version of `WdlDeclarationReference` (see my main comment) then I think we will want to look them up from different functions anyway so this might be a good thing to keep in place anyway.

Reply:
Going to the `Darcula` side.